### ISSUE TITLE
PTN000320364 

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#4695]
* **PTHR ID & PTN node:**

GO:0030050 | vesicle transport along actin filament | IBA with MGI:106612 , myo52 , 3143 , Q9Y4I1 , S000005853 , PTN000320364
-- | -- | --

[GO:0030050](https://www.pombase.org/term/GO:0030050)	
[vesicle transport along actin filament](https://www.pombase.org/term/GO:0030050)
[IBA](http://wiki.geneontology.org/index.php/Inferred_from_Biological_aspect_of_Ancestor_(IBA)) with [MGI:106612](http://www.informatics.jax.org/accession/MGI:106612) , [myo52](https://www.pombase.org/gene/SPCC1919.10c) , [3143](https://rgd.mcw.edu/rgdweb/search/search.html?term=3143) , [Q9Y4I1](https://www.uniprot.org/uniprot/Q9Y4I1) , [S000005853](https://www.yeastgenome.org/locus/S000005853) , [PTN000320364](http://www.pantherdb.org/panther/lookupId.jsp?id=PTN000320364)

* **Sequences with problematic annotation (ID + gene/protein name):**

https://www.pombase.org/gene/SPBC146.13c
(I think the process of transport along actin filament  might be for type V and/or Type V myosisns. This is a type I myosine which in pombe is restricted to endocytosis and does not have this role.

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**

Propagation?

* Add the label 'high priority' if needed. Generally high priority issues affect a lot of proteins, and annotations are incorrect (as opposed to just imprecise).

**Comment by ValWood** [comment]
Actually type one myosin also has a role in fission yeast used in oscillatoy movement of the nucleus, but as far as I'm aware, it doesn't have any role in the transport of vesicles along actin

**Comment by ValWood** [comment]
ignore!

**Comment by ValWood** [comment]
Although I would question this association:



GO:0031982 | vesicle | IBA with PTN000320364 , O43795 , MGI:105976 , Q29122 , DDB_G0289117 , DDB_G0272112 , FBgn0011225 , Q12965 , Q9UM54 , Q9LHE9 , 1560646 , Q9I8D1 , F1PRN2 , S000005853 , DDB_G0274575 , 621347 , MGI:104785 , Q9Y4I1 , MGI:104510 , DDB_G0288679 , MGI:106612 , 3143 , locus:2149932 , 621321
-- | -- | --


[GO:0031982](https://www.pombase.org/term/GO:0031982)	
[vesicle](https://www.pombase.org/term/GO:0031982)
[IBA](http://wiki.geneontology.org/index.php/Inferred_from_Biological_aspect_of_Ancestor_(IBA)) with [PTN000320364](http://www.pantherdb.org/panther/lookupId.jsp?id=PTN000320364) , [O43795](https://www.uniprot.org/uniprot/O43795) , [MGI:105976](http://www.informatics.jax.org/accession/MGI:105976) , [Q29122](https://www.uniprot.org/uniprot/Q29122) , [DDB_G0289117](http://dictybase.org/gene/DDB_G0289117) , [DDB_G0272112](http://dictybase.org/gene/DDB_G0272112) , [FBgn0011225](http://flybase.org/reports/FBgn0011225.html) , [Q12965](https://www.uniprot.org/uniprot/Q12965) , [Q9UM54](https://www.uniprot.org/uniprot/Q9UM54) , [Q9LHE9](https://www.uniprot.org/uniprot/Q9LHE9) , [1560646](https://rgd.mcw.edu/rgdweb/search/search.html?term=1560646) , [Q9I8D1](https://www.uniprot.org/uniprot/Q9I8D1) , [F1PRN2](https://www.uniprot.org/uniprot/F1PRN2) , [S000005853](https://www.yeastgenome.org/locus/S000005853) , [DDB_G0274575](http://dictybase.org/gene/DDB_G0274575) , [621347](https://rgd.mcw.edu/rgdweb/search/search.html?term=621347) , [MGI:104785](http://www.informatics.jax.org/accession/MGI:104785) , [Q9Y4I1](https://www.uniprot.org/uniprot/Q9Y4I1) , [MGI:104510](http://www.informatics.jax.org/accession/MGI:104510) , [DDB_G0288679](http://dictybase.org/gene/DDB_G0288679) , [MGI:106612](http://www.informatics.jax.org/accession/MGI:106612) , [3143](https://rgd.mcw.edu/rgdweb/search/search.html?term=3143) , [locus:2149932](http://arabidopsis.org/servlets/TairObject?accession=locus:2149932) , [621321](https://rgd.mcw.edu/rgdweb/search/search.html?term=621321)

Because although the myosin associates with the endosome, it isn't part of it?

**Comment by marcfeuermann** [comment]
A huge family. I've removed annotations to the root and re-annotated the family.
